src/main/java/uk/gov/justice/probation/courtcaseservice/service/CaseCommentsService.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.probation.courtcaseservice.service;
+
+import org.springframework.stereotype.Service;
+import uk.gov.justice.probation.courtcaseservice.jpa.entity.CaseCommentEntity;
+import uk.gov.justice.probation.courtcaseservice.jpa.repository.CaseCommentsRepository;
+import uk.gov.justice.probation.courtcaseservice.jpa.repository.CourtCaseRepository;
+import uk.gov.justice.probation.courtcaseservice.service.exceptions.EntityNotFoundException;
+
+import java.util.UUID;
+
+@Service
+public class CaseCommentsService {
+
+    private final CourtCaseRepository courtCaseRepository;
+    private final CaseCommentsRepository caseCommentsRepository;
+
+    public CaseCommentsService(CourtCaseRepository courtCaseRepository, CaseCommentsRepository caseCommentsRepository) {
+        this.courtCaseRepository = courtCaseRepository;
+        this.caseCommentsRepository = caseCommentsRepository;
+    }
+
+    public CaseCommentEntity createCaseComment(String caseId, CaseCommentEntity caseCommentEntity) {
+
+        CaseCommentEntity entity = caseCommentEntity.withCommentId(UUID.randomUUID().toString());
```

Comment:
Could use val 
```suggestion
       val entity = caseCommentEntity.withCommentId(UUID.randomUUID().toString());
```

Reply:
Typo, will update with `var`

Reply:
I actually asked myself which was right and got it wrong 😆 Confused by Kotlin 🙄